src/cpp/fastdds/builtin/type_lookup_service/TypeLookupReplyListener.cpp

```diff
@@ -18,17 +18,18 @@
  */
 
 #include <fastdds/builtin/type_lookup_service/TypeLookupReplyListener.hpp>
-#include <fastdds/builtin/type_lookup_service/TypeLookupManager.hpp>
-#include <fastdds/rtps/participant/RTPSParticipantListener.h>
-#include <fastdds/rtps/builtin/BuiltinProtocols.h>
 
+#include <fastrtps/rtps/history/ReaderHistory.h>
+#include <fastrtps/rtps/reader/RTPSReader.h>
 #include <fastrtps/types/TypeObjectFactory.h>
+#include <rtps/participant/RTPSParticipantImpl.h>
 
-#include <fastrtps/rtps/reader/RTPSReader.h>
-#include <fastrtps/rtps/history/ReaderHistory.h>
 #include <fastdds/dds/log/Log.hpp>
```

Comment:
This header is public and should be included in the block above (sorted alphabetically).

Reply:
[f288915](https://github.com/eProsima/Fast-DDS/pull/4053/commits/f288915aa343eabd6476d3875ea51d994ebe899d)